suse_migration_services/prechecks/repos.py

```diff
@@ -19,6 +17,7 @@ def remote_repos():
             log.error('No repositories in {}'.format(repos_path))
             return
 
+        remote_prefixes = ('http', 'ftp', 'plugin:/susecloud')
```

Comment:
This is by definition incomplete. There is nothing special about `plugin:/susecloud`. Anyone can write a URL resolver plugin for zypper and that plugin can resolve the prefix into a remote repo (http or ftp access) or a local repo. As such it would probably be good to have a comment here or find a name for the variable that is more expressive of it's intend.

Reply:
I agree, some naming that points to the fact that we accept repos of this type as "eligible to be used in a migration" would imho express the intend better

Reply:
Updated, between `migration_accepted` or `migration_safe` I chose the former